fb_reels_publishing_api_sample/README.md

```diff
@@ -48,6 +42,23 @@ Included in this repo is a sample video (located /sample_media) to be used durin
 
 ## Running the project
 
+### Using a [devcontainer](https://code.visualstudio.com/docs/devcontainers/containers)
```

Comment:
Could you add a statement that explicitly says that this works only on VSCode downloaded directly from the website and not a custom modified VSCode?

Reply:
What do you think about the note that I added to both readme files?